common/test/acceptance/tests/discussion/test_cohort_management.py

```diff
@@ -291,11 +302,152 @@ def test_add_new_cohort(self):
+
+        Given I have a course with a user in the course
+        When I add a new cohort with random assignment type to the course via the LMS instructor dashboard
+        Then the new cohort is displayed and has no users in it
+        And assignment type of displayed cohort is "random"
+        And when I add the user to the new cohort
+        Then the cohort has 1 user
+        And appropriate events have been emitted
+        Then I select the cohort (that you just created) from existing cohorts
+        Then I change its name and assignment type set to "manual"
+        Then I Save the settings
+        And cohort with new name is present in cohorts dropdown list
+        And cohort assignment type should be "manual"
+        """
+        cohort_name = str(uuid.uuid4().get_hex()[0:20])
+        new_cohort_name = '{old}__NEW'.format(old=cohort_name)
+        self._verify_cohort_settings(
+            cohort_name=cohort_name,
+            assignment_type='random',
+            new_cohort_name=new_cohort_name,
+            new_assignment_type='manual',
+            verify_updated=True
+        )
+
+    def test_update_existing_cohort_settings_with_empty_cohort_name(self):
+        """
+        Scenario: Update existing cohort settings(cohort name, assignment type).
+
+        Given I have a course with a user in the course
+        When I add a new cohort with random assignment type to the course via the LMS instructor dashboard
+        Then the new cohort is displayed and has no users in it
+        And assignment type of displayed cohort is "random"
+        And when I add the user to the new cohort
+        Then the cohort has 1 user
+        And appropriate events have been emitted
+        Then I select a cohort from existing cohorts
+        Then I set its name as empty string and assignment type set to "manual"
+        And I click on Save button
+        Then I should see an error message
+        """
+        cohort_name = str(uuid.uuid4().get_hex()[0:20])
+        new_cohort_name = ''
+        self._verify_cohort_settings(
+            cohort_name=cohort_name,
+            assignment_type='random',
+            new_cohort_name=new_cohort_name,
+            new_assignment_type='manual',
+            verify_updated=True
+        )
+
+    def test_default_cohort_assignment_settings(self):
+        """
+        Scenario: Cohort assignment settings are disabled for default cohort.
+
+        Given I have a course with a user in the course
+        And I have added a manual cohort
+        And I have added a random cohort
+        When I select the random cohort
+        Then cohort assignment settings are disabled
+        """
+        self.cohort_management_page.select_cohort("AutoCohort1")
+        self.cohort_management_page.select_cohort_settings()
+
+        self.assertTrue(self.cohort_management_page.is_assignment_settings_disabled)
+
+        message = "There must be one cohort to which students can be randomly assigned."
```

Comment:
Please change "randomly" to "automatically" and move "be" before "assigned": "There must be one cohort to which students can automatically be assigned."